packages/server-side-graphql-client/README.md

```diff
@@ -0,0 +1,392 @@
+<!--[meta]
+section: api
+title: Server-side graphQL client
+[meta]-->
+
+# Server-side graphQL client
+
+A library for running server-side graphQL queries and mutations in Keystone. It replaces the `keystone.createItems` method with a set of utility functions to generate and execute graphQL queries.
```

Comment:
```suggestion
A library for running server-side graphQL queries and mutations in Keystone.
```

Let's not leave references to `createItems` in here. It's really weird to have docs reference historical things that no longer exist, particularly for new users who have never seen (and will never see) `keystone.createItems`. Let's describe this package in its own right.

Reply:
ok